happymonk-ai/Ckdir-frontend-
# book demo page implementation

What does this PR do?
• Implement "Book DEMO" UI

How should this be manually tested?
• Clone this repo
• Check out this branch ft-homepage-update
• Run yarn install to install all packages
• Run yarn dev to start the development environment
• Navigate to http://localhost:3000/register to see the UI
